composer.json

```diff
@@ -21,8 +21,14 @@
             "Vimeo\\": "src/Vimeo"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "Vimeo\\VimeoTest\\": "tests/Vimeo"
```

Comment:
Change `Vimeo\\VimeoTest\\` to just `Vimeo\\` once the test is in just the `Vimeo` namespace.